cram-14.04.rosinstall

```diff
@@ -3,40 +3,16 @@
 - git:
     local-name: cram
     uri: https://github.com/cram2/cram.git
-    version: master
+    version: lecture-deps
 
 ############## ROS Lisp libraries, e.g. TF ###########
 - git:
     local-name: roslisp_common
     uri: https://github.com/ros/roslisp_common.git
     version: 093c131b6dcfa67f125fb3989eee14b5ba622bcf
-
-############## IAI semantic maps repository ##########
-- git:
-    local-name: iai_maps
-    uri: https://github.com/code-iai/iai_maps.git
-    version: 6a4f3fab1fa836fdf26e7ea9a36ab140c2e2d563
-
-############## IAI common msgs repository ############
+    
+############## IAI services and messages ###########
 - git:
-    local-name: iai_common_msgs
+    local-name: iai-messages
```

Comment:
Please rename this back to iai_common_msgs and do use the version, as the messages can change and break compilation.